client/src/components/Header/styles.js

```diff
@@ -5,10 +5,28 @@ export const GameTitle = styled.div`
   font-size: 26px;
   line-height: 32px;
   color: #ffffff;
+  position: sticky;
 `;
 
 export const CategoryTree = styled.div`
+  display: flex;
   font-family: "Motiva Sans", sans-serif;
   font-size: 12px;
   color: #56707f;
 `;
+
+export const Category = styled.div`
+  padding: 0px 3px;
+  &:hover {
```

Comment:
Solid on hover implementation, I've been using a lot of cursor changes on hover too so cool to see that you are also